src/pages/_document.tsx

```diff
@@ -46,6 +46,29 @@ class CustomDocument extends Document {
             sizes="16x16"
             type="image/png"
           />
+          <meta property="og:url" content="https://scribedao.com/" />
+          <meta property="og:type" content="website" />
+          <meta property="og:title" content="ScribeDAO" />
+          <meta
+            property="og:description"
+            content="Curating and simplifying the endless stream of crypto content."
+          />
+          <meta
+            property="og:image"
+            content="https://scribedao.vercel.app/static/favicons/apple-touch-icon-120x120.png"
+          />
+
+          <meta name="twitter:card" content="summary_large_image" />
+          <meta property="twitter:url" content="https://scribedao.com/" />
+          <meta name="twitter:title" content="ScribeDAO" />
+          <meta
+            name="twitter:description"
+            content="Curating and simplifying the endless stream of crypto content."
+          />
+          <meta
+            name="twitter:image"
+            content="https://scribedao.vercel.app/static/favicons/apple-touch-icon-120x120.png"
```

Comment:
I have had issues in past with twitter when just using relative path. Also this is just a placeholder need a bigger image `1200 x 630`